release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1773,6 +1773,16 @@ With this release, availability sets are no longer modified after creation, allo
 [id="ocp-release-note-machine-config-operator-bug-fixes_{context}"]
 ==== Machine Config Operator
 
+* Previously, the `ContainerRuntimeConfig` incorrectly set the `--root` path for the `runc` runtime. This caused containers to run with an incorrect root path and led to issues with the container operations. With this release, the `--root` path for the container runtime is correct and matches the specified runtime, providing consistent operation. (link:https://issues.redhat.com/browse/OCPBUGS-47629[OCPBUGS-47629])
+
+* Previously, if you launched a cluster for an {aws-first} region for which an Amazon Machine Images (AMI) is not available, the installation might fail. This was problematic for OKD, which does not have an AMI for all of the AWS regions. With this release, an update adds a default region that is used in case the cluster is launched with a region is not available. (link:https://issues.redhat.com/browse/OCPBUGS-55231[OCPBUGS-55231])
```

Comment:
```suggestion
* Previously, if you launched a cluster for an {aws-first} region for which an Amazon Machine Images (AMI) is not available, the installation might fail. This was problematic for OKD, which does not have an AMI for all of the AWS regions. With this release, an update adds a default region that is used in case the cluster is launched with a region is not available. (link:https://issues.redhat.com/browse/OCPBUGS-55231[OCPBUGS-55231])
```
We have a release note for OKD in the OpenShift docs? Is this correct?

Reply:
Removed this entry and msg'ed the component writer...